# 3D Touching Point Pair
geometry_msgs/Point point1
geometry_msgs/Point point2